templates/kubernetes-cluster.template

```diff
@@ -667,8 +669,9 @@ Resources:
               command: "systemctl enable awslogs.service && systemctl start awslogs.service"
             # This joins the existing cluster with kubeadm.  Kubeadm is preinstalled in the AMI this template uses, so
             # all that's left is to join the cluster with the correct token.
+            # TODO(chuckha) stop using --discovery-token-unsafe-skip-ca-verification
             "03-k8s-setup-node":
-              command: !Sub "kubeadm reset && kubeadm join --node-name=\"$(hostname -f)\" --token=${KubeadmToken.Token} ${K8sMasterInstance.PrivateIp}:6443"
+              command: !Sub "kubeadm reset && kubeadm join --node-name=\"$(hostname -f)\" --token=${KubeadmToken.Token} --discovery-token-unsafe-skip-ca-verification ${K8sMasterInstance.PrivateIp}:6443"
```

Comment:
@mattmoyer said this is ok for now and gave some suggestions on how to improve this. I'd like to fix this by release time.

https://kubernetes.io/docs/reference/setup-tools/kubeadm/kubeadm-join/#discovering-what-cluster-ca-to-trust

URL discovery is the method we will likely use, but I'll explore other options that are less of a deviation from what we're doing already.